CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 19/02/2020
+- Add endpoint for finding subscriptions by id - can only be called by other MSs.
```

Comment:
remove `can only be called by other MSs.` -> the purpose of the changelog is to highlight changes, not document behavior